lib/boot.js

```diff
@@ -1,11 +1,13 @@
-var Hub = require('./hub')
+var config = require('./../config/config')
+var Hub = require('./hub/hub')
 var co = require('co')
 
 co(function*() {
-  var hub = new Hub()
+  console.log("Starting ServiceHub")
+  var hub = new Hub({ config: config })
   yield hub.start()
```

Comment:
@igorshapiro Is `start` actually defined on the `Hub` class? I couldn't find it...